src/rocks_compaction_scheduler.h

```diff
@@ -63,5 +71,18 @@ namespace mongo {
         // We'll compact the prefix if any operation on the prefix reports more than 50.000
         // deletions it had to skip over (this is about 10ms extra overhead)
         static const int kSkippedDeletionsThreshold = 50000;
+
+        // thread for async execution of range compactions
+        bool _compactionThreadRunning = true;
+        boost::mutex _compactionMutex;
+        boost::condition_variable _compactionWakeUp;
+        std::deque<RocksCompactionOperation*> _compactionQueue;
+
+        void compactionThread();
+        boost::thread _compactionThread;
+
+        // schedule compact range operation for execution in _compactionThread
+        Status scheduleCompactOp(RocksCompactionOperation* cop);
```

Comment:
Private methods should go after public/protected methods and before data members.